crates/lib/src/qvm/mod.rs

```diff
@@ -147,6 +153,33 @@ pub fn apply_parameters_to_program(
     Ok(program)
 }
 
+/// Options avaialable for running programs on the QVM.
+#[allow(clippy::module_name_repetitions)]
+#[derive(Clone, Copy, Debug)]
+pub struct QvmOptions {
+    /// The timeout to use for requests to the QVM. If set to [`None`], there is no timeout.
+    pub timeout: Option<Duration>,
+}
+
+impl QvmOptions {
+    /// Builds a [`QvmOptions`] with the zero value for each option.
+    ///
+    /// Consider using [`Default`] to get a reasonable set of
+    /// configuration options as a starting point.
+    #[must_use]
+    pub fn new() -> Self {
+        Self { timeout: None }
+    }
+}
+
+impl Default for QvmOptions {
+    fn default() -> Self {
+        Self {
+            timeout: Some(Duration::from_secs_f32(DEFAULT_QVM_TIMEOUT)),
```

Comment:
Goes with the above suggestion

```suggestion
            timeout: Some(DEFAULT_QVM_TIMEOUT),
```